src/products/mozilla_org/testing.md

```diff
@@ -73,27 +73,24 @@ Updated translations are pushed to the production server regularly throughout th
 
 It is safe to say that it will take less than an hour to see your changes reflected on the production server.
 
-When a project has a firm deadline to meet, it will be communicated through the [dev-l10n-web mailing list](https://lists.mozilla.org/listinfo/dev-l10n-web). Be sure to sign up so you receive important community wide information on web related projects. You can also check out the deadline at page level in Pontoon.
+When a project has a firm deadline to meet, it will be indicated at the top left corner of the project page in Pontoon and at the page level. When necessary, it will be communicated through [the l10n-community room on Matrix](https://chat.mozilla.org/#/room/#l10n-community:mozilla.org).
 
-## Testing content migrated to Fluent
-
-This section focuses on review and test after a page is migrated to Fluent.
+## Testing content in Fluent
 
 ### Reviewing and updating [brands.ftl](https://github.com/mozilla-l10n/www-l10n/blob/master/en/brands.ftl)
 
-This new file captures all the brand and product names that appear in all the pages on mozilla.org as of now. As a general rule, brands and product names remain unchanged. The exceptions are:
+This file captures all the brand and product names that appear in all the pages on mozilla.org. As a general rule, brands and product names remain unchanged. The exceptions are:
 
-* Account/Accounts in Firefox Account/Accounts
-* Sync in Firefox Sync
-* Browser in Firefox Browser
+* Account/accounts in Mozilla account/accounts: Mozilla should be treated as a brand, but `account` and `accounts` should be translated. Both the singular and plural forms are used in lower cases. Uppercase is reserved for special use cases where headline case is necessary, for example legal document names and references.
+* Browser/Browsers in Firefox Browser/Browsers
 
 If your community decides to translate any of these products or feature names, translate it once and translate it in this file. Be sure to have a consensus among active contributors what the translation should be before updating this file. This practice will ensure term use consistency across all pages.  
 
 Please refer to the [guidelines for brands and trademarks](https://mozilla-l10n.github.io/styleguides/mozilla_general/index.html#brands-copyright-and-trademark) when reviewing and updating this file.
 
 ### Reviewing and updating migrated files
```

Comment:
Do you think this section still relevant, given how many years passed from the migration?

If yes, then you need to explain what "migrated" means (from what? when?).

Reply:
I removed this section after reviewing the few locales still have errors from the migration. They have extremely low completion and are mostly inactive.